scripts/netlify-server.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+'use strict';
+
+/* run a local server to edit content in the cms
+  Make sure to run `npm run build` to build the dist for starting point
+*/
+const program = require('commander');
+
+// const fsExpressAPI = require('./src/lib/fs-api/middleware');
+const express = require('express');
+const path = require('path');
+const projectRoot = path.join(__dirname, "../");
+
+program
+  .version('0.1.0')
+  .description('Run the express server')
+  .usage('<cmd>')
+  .option('-p, --cms-path <cmsPath>', 'CMS Absolute Path', '/')
+  .option('-l, --site-location <siteLocation>', 'Site File Location', 'test')
+  .option('-c, --cms-location <cmsLocation>', 'CMS File Location', 'test')
+  .option('-d, --dist-location <distLocation>', 'CMS Build Location', 'dist')
+  .action((cmd, command) => {
+    // console.log('cmd', cmd);
+    console.log(`Using ${ command.cmsPath } for absolute url path.`);
```

Comment:
Use a single `console.log` call with a template string to retain formatting.

Reply:
Will also make for a cleaner output. 👍